client/src/components/main/coffee_shop/CoffeeShopIndex.scss

```diff
@@ -57,10 +36,12 @@
   width: 100%;
   max-width: 90%;
   ul {
+    width: 100%;
     list-style: none;
     display: flex;
+    align-items: stretch;
+    flex-direction: column;
     justify-content: flex-start;
-    flex-wrap: wrap;
```

Comment:
What's the reasoning behind removing the `flex-wrap` here?

Reply:
Well, I figured the behavior the `flex-wrap` was achieving here is essential the same as the `flex-direction: column` and I thought doing it this way seemed to make a little more sense semantically. I may be wrong but I'm pretty sure the end result is identical.

Reply:
Actually it seems that I was mistaken. I forgot that the tiles were being displayed in a grid, that's my mistake for only looking at the changes on a narrow screen, oops

Reply:
Yeah the reason I ask is b/c I believe I remember putting this in because the elements didn't work on all sizes. No worries.

Reply:
@jmsteen The latest commit fixes this, thanks for noticing that 💖

Reply:
No prob! Sorry, I have a few more questions I'm going to post on other comments